notebooks/Beginners_guide/08_Zonal_statistics.ipynb

```diff
@@ -104,7 +109,9 @@
     "\n",
     "To upload a file from your computer, open the directory browser, drag your file in the `uploads` directory and drop it (when a <span style=\"display:inline-block;heigth:20px;width:20px;border-radius:10px;background-color:#50b745;color:white;text-align:center\">+</span> symbol appears).\n",
     "\n",
-    "> **Before running the next cell:** upload the shapefile (i.e., several files) you would like to use in the `uploads` directory and indicate the filename of your shapefile in the cell."
+    "> **Before running the next cell:** upload the shapefile (i.e., several files) you would like to use in the `uploads` directory and indicate the filename of your shapefile in the cell.\n",
```

Comment:
I don't think this will work as everything inside the top level 'resources' folder is read-only. Users will have to create their own 'uploads' folder in their own space